src/routes/outcomes/TreeNode.svelte

```diff
@@ -44,8 +44,8 @@
 	{#if showInfo}
 		<div>
 			<p>
-				{node.explanation} PauseAI estimates this probability at roughly
-				<b>{node.probability * 100}%</b>.
+				{node.explanation} <!--PauseAI estimates this probability at roughly
```

Comment:
I think we could add an if condition that checks the `type` of the node, and conditionally prints the `doomer_help_str` or `optimist_help_str`. It would be cleaner than adding the text to the source json.

Reply:
I prefer the particular message that the safe node in case of a pause has. Ideally they should all be personalized.